api/v1/submissions/PKPSubmissionHandler.php

```diff
@@ -170,6 +173,11 @@ public function __construct()
                     'handler' => [$this, 'getContributor'],
                     'roles' => [Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR, Role::ROLE_ID_ASSISTANT, Role::ROLE_ID_REVIEWER, Role::ROLE_ID_AUTHOR],
                 ],
+                [
+                    'pattern' => $this->getEndpointPattern() . '/{submissionId:\d+}/reviewAssignments/{reviewAssignmentId:\d+}',
+                    'handler' => [$this, 'getReviewAssignment'],
+                    'roles' => [Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR],
```

Comment:
What about assistants? Do they have access to the review assignments if they are assigned to the review stage?